Simplified the donate for 2020

jeremyzilar opened the pull request:
This change makes the Donate page into a basic donation page.
It removes the copy associated with the end-of-year campaign.

![image](https://user-images.githubusercontent.com/395641/71785864-b6849000-2fd2-11ea-9942-9071f1d5f7a5.png)

afeld commented:
Is this an alternative to https://github.com/brooklynrail/brooklynrail-support/pull/23?

jeremyzilar commented:
Yes, in https://github.com/brooklynrail/brooklynrail-support/pull/23 
I removed the donate function and made the page about "saying thanks".

In this PR, the donate function is still there. I just removed the end-of-year campaign copy.
The page remains a low-friction way to donate.
